README.md

```diff
@@ -125,6 +125,8 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 
 ## Open Source books
 - [Producing Open Source Software](https://producingoss.com/) - Producing Open Source Software is a book about the human side of Open Source development. It describes how successful projects operate, the expectations of users and developers, and the culture of free software.
+- [Pro Git (Second Edition)](https://www.freetechbooks.com/pro-git-second-edition-t1202.html) - A fully-updated guide to Git and its usage in the modern world. This book teaches readers how to master the world of distributed version workflow, use the distributed features of Git to the full, and extend Git to meet their every need.
+- [The Architecture of Open Source Applications, Volume II: Structure, Scale, and a Few More Fearless Hacks](https://www.freetechbooks.com/the-architecture-of-open-source-applications-volume-ii-structure-scale-and-a-few-more-fearless-hacks-t1266.html) -The authors of twenty-four open source applications explain how their software is structured, and why. From web servers and compilers to health record management systems, they are covered here to help you become a better developer.
```

Comment:
Part of this is already in our list, but shown as just the git tool. We can update this link to point to all the books that are already freely available.

```suggestion
- [The Architecture of Open Source Applications](https://www.aosabook.org/en/index.html) - The authors of twenty-four open source applications explain how their software is structured, and why. From web servers and compilers to health record management systems, they are covered here to help you become a better developer.
```